padl/transforms.py

```diff
@@ -861,11 +913,25 @@ def infer_apply(self, inputs):
         :param inputs: The input.
         """
         self.pd_forward_device_check()
+        in_args = inputs
+        _pd_trace.clear()
         inputs = self.pd_preprocess.pd_call_transform(inputs, mode='infer')
         inputs = _move_to_device(inputs, self.pd_device)
-        inputs = self.pd_forward.pd_call_transform(inputs, mode='infer')
-        inputs = self.pd_postprocess.pd_call_transform(inputs, mode='infer')
-        return self._pd_format_output(inputs)
+        try:
+            inputs = self.pd_forward.pd_call_transform(inputs, mode='infer')
+        except Exception as err:
+            self._pd_trace_error(self._pd_get_error_idx('forward'), in_args)
+            raise err
+        try:
+            inputs = self.pd_postprocess.pd_call_transform(inputs, mode='infer')
+        except Exception as err:
+            self._pd_trace_error(self._pd_get_error_idx('postprocess'), in_args)
+            raise err
+        output_format = self._pd_get_output_format()
```

Comment:
Could you change the last few lines to `return self._pd_format_output(inputs)`? This is causing a lot of the tests to fail currently.

Reply:
thanks! nice catch, i think i changed that recently by mistake